Gruntfile.js

```diff
@@ -97,6 +97,7 @@ module.exports = function run(grunt) {
     grunt.registerTask('serve', 'Start development web server', ['build', 'copy-test', 'connect', 'watch']);
     grunt.registerTask('serve-nobuild', 'Start development web server on current build (USE WITH CAUTION, only use with known state of directories dist and tmp)', ['nobuild-warning', 'connect', 'watch']);
     grunt.registerTask('test', 'Start dev web server and run tests', ['setenv', 'build-prod', 'copy-test', 'connect', 'webdriver:maintests', 'test-links']);
+    grunt.registerTask('test-nobuild', 'Run tests on current build (USE WITH CAUTION, only use with known state of directories dist and tmp)', ['nobuild-warning', 'setenv', 'copy-test', 'connect', 'webdriver:maintests', 'test-links']);
 
     grunt.registerTask('nobuild-warning', 'Issue a warning on screen about using serve-nobuild', () => {
         grunt.log.writeln('***** WARNING ***** When using "serve-nobuild", you have to be sure that the directories "dist" and "tmp" are in a known good state (as they would be after a build)');
```

Comment:
Just to be explicit...

```suggestion
        grunt.log.writeln('***** WARNING ***** When using "serve-nobuild" or "test-nobuild", you have to be sure that the directories "dist" and "tmp" are in a known good state (as they would be after a build)');
```

Reply:
fixed :)